src/views/admin/PersonManagementView.vue

```diff
@@ -16,16 +22,44 @@
     { title: t('person.lastName'), key: 'person.name.familienname', align: 'start' },
     { title: t('person.firstName'), key: 'person.name.vorname', align: 'start' },
     { title: t('person.userName'), key: 'person.referrer', align: 'start' },
+    { title: t('person.rolle'), key: 'rolle', align: 'start' },
+    { title: t('person.zuordnung'), key: 'administrationsebenen', align: 'start' },
   ];
 
+  type PersonenWithRolleAndZuordnung = {
+    rolle: string;
+    administrationsebenen: string;
+    person: Person;
+  }[];
   const router: Router = useRouter();
 
   function navigateToPersonDetails(_$event: PointerEvent, { item }: { item: Personendatensatz }): void {
     router.push({ name: 'person-details', params: { id: item.person.id } });
   }
 
+  // Maps over allPersons, finds the corresponding zuordnungen for each person by matching the personId, and then extracts and combines
+  // the rolle values into a single comma separated string. The result is merged with the original person data, adding a new role property.
+  const personenWithUebersicht: ComputedRef<PersonenWithRolleAndZuordnung> = computed(() => {
+    return personStore.allPersons.map((person: Personendatensatz) => {
+      const uebersicht: Uebersicht = personenKontextStore.allUebersichte?.items.find(
+        (ueb: Uebersicht) => ueb?.personId === person.person.id,
+      );
+      const rollen: string = uebersicht?.zuordnungen.length
+        ? uebersicht.zuordnungen.map((zuordnung: Zuordnung) => zuordnung.rolle).join(', ')
+        : '-';
+      // Choose sskDstNr if available, otherwise sskName.
+      const administrationsebenen: string = uebersicht?.zuordnungen.length
+        ? uebersicht.zuordnungen
+            .map((zuordnung: Zuordnung) => (zuordnung.sskDstNr ? zuordnung.sskDstNr : zuordnung.sskName))
+            .join(', ')
+        : '-';
+      return { ...person, rolle: rollen, administrationsebenen: administrationsebenen };
```

Comment:
```suggestion
      return { ...person, rollen, administrationsebenen };
```
In "rollen" umbenennen, dann können wir den shorthand benutzen. Es sollen ja zukünftig eh mehrere angezeigt werden